specification/src/main/asciidoc/platform/BrandComparison.adoc

```diff
@@ -0,0 +1,170 @@
+|Jakarta™ RESTful Web Services 2.1
+
+|Java API for JSON Processing 1.1
+|Jakarta™ JSON Processing 1.1
+
+|Java API for JSON Binding 1.0
+|Jakarta™ JSON Binding 1.0
+
+|Java™ Platform, Enterprise Edition Management 1.1
+|Jakarta™ Management 1.1
+
+|Java™ Platform, Enterprise Edition Deployment 1.2
+|Jakarta™ Deployment 1.7
+
+|Java™ Authorization Service Provider Contract for Containers 1.5
+|Jakarta™ Authorization 1.5
+
+|Java™ Authentication Service Provider Interface for Containers 1.1
+|Jakarta™ Authentication 1.1
+
+|Java™ EE Security API 1.0
+|Jakarta™ Security 1.0
+
+|Java Persistence 2.2
+|Jakarta™ Persistence 2.2
+
+|Bean Validation 2.0
+|Jakarta™ Bean Validation 2.0
+
+|Managed Beans 1.0
+|Jakarta™ Managed Beans 1.0
+
+|Interceptors 1.2 rev A
+|Jakarta™ Interceptors 1.2
+
+|Contexts and Dependency Injection for the Java EE Platform 2.0
+|Jakarta™ Contexts and Dependency Injection 2.0
+
+|Dependency Injection for Java 1.0
+|Jakarta™ Dependency Injection 1.0
+
+|Concurrency Utilities for Java EE 1.0
+|Jakarta™ Concurrency 1.1
+
+|Batch Applications for the Java Platform 1.0 rev A
+|Jakarta™ Batch 1.0
+|===
+
+=== Transaction 1.2 vs 1.3
+
+To address cleanly separating the `javax.transaction` package in the JVM from the classes being donated, a Maintenance Release of the Java™ Transaction API (JTA) was created in the JCP and released as version 1.3.  The official Jakarta version is therefore 1.3 and not 1.2.
+
+No API changes were made in this Maintenance Release or after donation and Java™ Transaction API 1.2, Java™ Transaction API 1.3 and Jakarta Transaction 1.3 are functionally equivalent.
+
+=== Activation 1.1 vs 1.2
+
+The Java EE 8 Platform specification, dated July 31st, 2017, references JavaBeans™ Activation Framework Specification Version 1.1 in appendix EE.C Related Documents.  A Maintenance Release of the JavaBeans™ Activation Framework was created and released in the JCP as version 1.2 dated July 2017.  The reference to JAF 1.1 in the Java EE 8 Platform specification, released at the same time is simply an error.
+
+The official Java EE 8 version of JAF is 1.2 and therefore the official version for Jakarta EE 8 is Jakarta Activation 1.2.
+
+JavaBeans™ Activation Framework 1.1 and 1.2 are distinct specifications and may be functionally different.  JavaBeans™ Activation Framework 1.2 and Jakarta Activation 1.2 are functionally equivalent.
+
+=== Web Services Metadata 2.1 vs 1.1
+
+In JSR-000181 Web Services Metadata for the JavaTM Platform Maintenance Release 2, the specification version was mistakenly incremented from 1.0 to 2.1.  No binaries were released with this version and was only an error of documentation, therefore the version was corrected after donation.
+
+The official version of Web Services Metadata for Jakarta EE 8 is version 1.1, which is functionally equivalent to Web Services Metadata 2.1 in Java EE 8.
+
+===  SOAP with Attachments 1.3 vs 1.4
+
+TODO: I do not know the explanation for this one
```

Comment:
I think it would be good to leave this as-is.  An "easter egg" type of mistake.  :-)